python/ray/tune/tuner.py

```diff
@@ -301,6 +314,10 @@ def train_fn(config):
         Returns:
             bool: True if this path exists and contains the Tuner state to resume from
         """
+        if _use_storage_context():
+            fs, fs_path = get_fs_and_path(path, storage_filesystem)
+            return _TUNER_PKL in _list_at_fs_path(fs, fs_path)
```

Comment:
I think it would be more efficient to use an exists() call instead of a list operation.